### PR TITLE
Update github.com/containerd/containerd to v1.5.13 to fix security issues

### PR DESCRIPTION
**Pull request by bmelbourne**
Update Go github.com/containerd/containerd package to **v1.5.13** to fix critical security vulnerabilities outlined in CVE-2021-43816 and CVE-2022-31030.

https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-43816
https://github.com/containerd/containerd/security/advisories/GHSA-5ffw-gxpp-mxpf (CVE-2022-31030)

### CODE REVIEW

**Review by nyrahul — COMMENTED** [review]
Thanks for the PR, @bmelbourne
I am getting a different `go.sum` when I run `go mod tidy` with the new `go.mod`. This is been verified. We are also trying to add a CI check to cross-verify the generated `go.sum`.

**Comment by bmelbourne** [comment]
> Thanks for the PR, @bmelbourne I am getting a different `go.sum` when I run `go mod tidy` with the new `go.mod`. This is been verified. We are also trying to add a CI check to cross-verify the generated `go.sum`.

@nyrahul
Containerd have been releasing some rapid updates recently to fixes various issues, and I may have inadvertently forgotten to run `go mod tidy`. PR now updated.

File: KubeArmor/go.mod [hunk 1]
```diff
@@ -19,7 +19,8 @@ replace (
 )
 
 require (
-	github.com/containerd/containerd v1.5.2
+	github.com/Microsoft/go-winio v0.5.2 // indirect
```
Comment (nyrahul) [inline KubeArmor/go.mod]: Is there a reason to explicitly state this indirect dependency? Similarly, there are few other indirect dependencies down this file which are newly added and the compilation works fine even if removed.

Comment (nyrahul) [reply KubeArmor/go.mod]: An indirect dependency on MS winio! I am not sure if this is needed. Please check. Thanks

File: KubeArmor/go.sum [hunk 2]
```diff
@@ -169,8 +183,9 @@ github.com/containerd/ttrpc v0.0.0-20190828154514-0e0f228740de/go.mod h1:PvCDdDG
 github.com/containerd/ttrpc v0.0.0-20190828172938-92c8520ef9f8/go.mod h1:PvCDdDGpgqzQIzDW1TphrGLssLDZp2GuS+X5DkEJB8o=
 github.com/containerd/ttrpc v0.0.0-20191028202541-4f1b8fe65a5c/go.mod h1:LPm1u0xBw8r8NOKoOdNMeVHSawSsltak+Ihv+etqsE8=
 github.com/containerd/ttrpc v1.0.1/go.mod h1:UAxOpgT9ziI0gJrmKvgcZivgxOp8iFPSk8httJEt98Y=
-github.com/containerd/ttrpc v1.0.2 h1:2/O3oTZN36q2xRolk0a2WWGgh7/Vf/liElg5hFYLX9U=
 github.com/containerd/ttrpc v1.0.2/go.mod h1:UAxOpgT9ziI0gJrmKvgcZivgxOp8iFPSk8httJEt98Y=
+github.com/containerd/ttrpc v1.1.0 h1:GbtyLRxb0gOLR0TYQWt3O6B0NvT8tMdorEHqIQo/lWI=
```
Comment (nyrahul) [inline KubeArmor/go.sum]: When I run `go mod tidy` it does not generate this line. When I checked the details, this line is supposed to get the checksum of entire code base of ttrpc whereas the next line containing `....ttrpc/go.mod` simply gets a checksum on the `go.mod` file.
Point is, do we require this explicit entire code base checksum to be added?
Also I dont understand why it does not get added when I run `go mod tidy`. Is there any specific flag you used with the tidy command?

**Review by nyrahul — COMMENTED** [review]
Please find comments inline. Thanks again for the PR.

**Pull request closed without merging**